### DISCUSSION TITLE
Live

### DISCUSSION THREAD

**Discussion by charlotteinjp** [ChurchApps/ChurchAppsSupport#314] [FreeShow] [answered]
I was wondering if it were possible to add cropping to the live windows / screens output.
This would really help as often I only need to present a certain part of a window or screen, and I'm sure there are others that would also highly appriciate this improvement.

Thanks

**Comment by vassbo** [comment, marked as answer]
Awesome suggestion! Shouldn't be too hard to implement. I will see what I can do. 😊

**Comment by vassbo** [comment]
Cropping is added in v0.9.0! 😃
Apply a style to the output and change the cropping from the "Styles" settings tab.

If you want to hide certain parts of the output instead of resizing it, you should add an overlay with black boxes and lock it.
